.github/workflows/docker.yml

```diff
@@ -8,11 +8,11 @@ jobs:
 
   docker:
     name: build and push
-    runs-on: ubuntu-latest
+    runs-on: s1tbx-latest
```

Comment:
As it looked like a typo to me I have changed this line. Pls check @mmacata if that's right.

Reply:
This is a coincidence as our other branch is named like this. It is not a typo as this image referrs to where the build runs and the images are provided by github. We use ubuntu-latest to run the build, in this case an alpine build. Docker in docker :upside_down_face:

Reply:
I see, reverted in c8ac7e0